tests/product/base_product_case.py

```diff
@@ -496,11 +496,11 @@ def status_node_connection_error(self, host):
 
     def create_presto_client(self, host=None):
         ips = self.cluster.get_ip_address_dict()
+        config_path = os.path.join('~', LOCAL_CONF_DIR, COORDINATOR_DIR_NAME, CONFIG_PROPERTIES)
+        config = self.cluster.exec_cmd_on_host(self.cluster.master, 'cat ' + config_path)
```

Comment:
You could also use sudo (use_sudo=true or something like that) to retrieve the file (that's what presto-admin does internally) but this is also fine.